src/pages/index.js

```diff
@@ -123,6 +126,10 @@ class IndexPage extends React.Component {
                 />
               </ListItem>
             </List>
+
+            <div>
+                <MyComments />
```

Comment:
MyComments should show comments from any user, not only from the logged in user. Therefore, a new API is required like in banter.

Reply:
Loading comments via the model method `.fetchList` requires that the user is logged in. Therefore we need a new API like in banter: https://github.com/blockstack-radiks/banter/blob/master/server/ApiController.js#L15